packages/cli/package.json

```diff
@@ -23,17 +24,18 @@
   "devDependencies": {
     "@oclif/dev-cli": "^1.22.2",
     "@oclif/test": "^1.2.5",
+    "@types/adm-zip": "^0.4.33",
+    "@types/glob": "^7.1.3",
     "@types/js-yaml": "^3.12.1",
-    "@types/node": "8.10.34",
+    "@types/node": "10.12.0",
     "@types/yeoman-environment": "^2.3.2",
     "@types/yeoman-generator": "^3.0.2",
     "@types/yosay": "^0.0.29",
-    "@types/glob": "^7.1.3",
     "ts-node": "^8.4.1",
     "typescript": "^3.6.3"
   },
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=10.12.0"
```

Comment:
Please make sure you're documenting such important details as NodeJS version somewhere, the README maybe is a good place to note this information.

Reply:
This would be part of an larger effort to document the usage of the SDK